osu.Game/Beatmaps/Formats/IHasComboColours.cs

```diff
@@ -8,6 +8,19 @@ namespace osu.Game.Beatmaps.Formats
 {
     public interface IHasComboColours
     {
-        List<Color4> ComboColours { get; set; }
+        /// <summary>
+        /// Retrieves the list of combo colours for presentation only.
+        /// </summary>
+        IReadOnlyList<Color4> ComboColours { get; set; }
+
+        /// <summary>
+        /// Adds combo colours to the list.
+        /// </summary>
+        void AddComboColours(params Color4[] colours);
+
+        /// <summary>
+        /// Clear current combo colours from the list.
+        /// </summary>
+        void ClearComboColours();
```

Comment:
when would we every want to clear combo colours?

Reply:
Wasn't sure about leaving an add function without remove/clear function.., will remove.